route_following_plugin/src/route_following_plugin.cpp

```diff
@@ -92,139 +103,105 @@ namespace route_following_plugin
         if(last_lanelet_index == -1)
```

Comment:
A comment describing what the above loop is meant to do would be good to include